server/test/test_api.py

```diff
@@ -25,7 +25,7 @@ class EndPoints(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        cls.ps = Popen(["cellxgene", "launch", "--no-open", "example-dataset/pbmc3k.h5ad"])
+        cls.ps = Popen(["cellxgene", "launch", "example-dataset/pbmc3k.h5ad"])
```

Comment:
Maybe we should use the --debug flag for the tests so we get the full stack trace?

Reply:
sure